users_permissions/README.md

```diff
@@ -75,59 +75,45 @@ Then run this command:
+    The rules defined above allows the administrators to read/write the Items of all the users. The non-admin users will be able to read/write only their own Items. For all the other collections (like Roles for example) it is valid that users can change only the role that belongs to them. For this sample, this happens when the users register themselves.
+    1. Click the button `Save dtaft` and confirm.
+    1. Click the button `Review draft & deploy`.
+1. Go to the **Sync** menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
     1. Don't create a schema. Skip by choosing "No thanks, continue to Sync".
     1. Press the "Flexible Sync" button. Only Flexible Sync is supported in the Realm Flutter SDK.
     1. Switch ON the ["Development mode"](https://www.mongodb.com/docs/atlas/app-services/sync/data-model/development-mode/) option.
     1. Create a new database collection and choose a name for it.
     1. Create a new queryable field used for filtering data between both realms in this application. For our sample the field is `owner_id`.
         Type the field name `owner_id` in the selection box and then choose `Create owner_id`.
         It will be created.
-    1. Define permission - for the purpose of this sample please choose the option `Custom` and write the following rules:
-    ```json {
-    {
-      "rules": {
-        "Item": [
-          {
-            "name": "admin",
-            "applyWhen": {
-              "%%user.custom_data.isAdmin": true
-            },
-            "read": true,
-            "write": true
-          },
-          {
-            "name": "user",
-            "applyWhen": {
-              "%%user.custom_data.isAdmin": false
-            },
-            "read": true,
-            "write": {
-              "owner_id": "%%user.id"
-            }
-          }
-        ]
-      },
-      "defaultRoles": [
-        {
-          "name": "read-write",
-          "applyWhen": {},
-          "read": {
-            "owner_id": "%%user.id"
-          },
-          "write": {
-            "owner_id": "%%user.id"
-          }
-        }
-      ]
-    }
-   ```
-    The rules defined above allows the administrators to read/write the Items of all the users. The non-admin users will be able to read/write only their own Items. For all the other collections (like Roles for example) it is valid that users can change only the role that belongs to them. For this sample, this happens when the users register themselves.
-    1. Click the button `Enable Sync` and confirm.
-    1. Click the button `Review draft & deploy`, again.
 1. [Find and Copy the App ID](https://www.mongodb.com/docs/atlas/app-services/reference/find-your-project-or-app-id/) of your new application.
 1. Go to `\assets\atlas_app\realm_config.json` in this sample and set your app_Id as follow:
     ```json{
```

Comment:
cant edit the code example b/c not changed but needs to be updated to `app_id` (lowercase 'i' in 'id').

Reply:
also [on line 128](https://github.com/realm/realm-dart-samples/pull/44/files#diff-0f35b096cabe9a6266cb856f43be42c8e19590273707b8397769735ee10a3668R128), it says:

> Follow the instructions in the next section `Create administrator using Realm SDK`

this should be changed to:

```md
Follow the instructions in the section [Create administrator using Realm SDK](#create-administrator-using-realm-sdk)
```